src/test/java/pages/BasePage.java

```diff
@@ -0,0 +1,33 @@
+package pages;
+
+import org.openqa.selenium.*;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public abstract class BasePage {
+
+    protected static final int WAITING_PERIOD = 20;
```

Comment:
as for me, **TIMEOUT_IN_SECONDS** sounds more clear than **WAITING_PERIOD**....

Reply:
Agree